README.md

```diff
@@ -3,91 +3,70 @@
 
 ## Requirements
 
-- Python 3.x
-- Postgres >= 9
+- [Docker](https://www.docker.com/)
+
 
 ## Running the app locally
 
-Perform the following steps from your terminal.
+We use Docker for local development. To get started, run the following from your terminal:
 
 1. Clone this repository and `cd` into your local copy.
 
     ```bash
     git clone git@github.com:datamade/committee-oversight.git
     cd committee-oversight
     ```
-2. Create a virtual environment. (We recommend using [`virtualenvwrapper`](http://virtualenvwrapper.readthedocs.org/en/latest/install.html) for working in a virtualized development environment.)
 
-    ```bash
-    mkvirtualenv committee-oversight
-    ```
-3. Install the requirements.
+2. Load initial category data:
 
     ```bash
-    pip install -r requirements.txt
+    docker-compose -f docker-compose.yml -f docker-compose.db-ops.yml run --rm dbload
     ```
 
-4. Copy the example local settings file to the correct location:
+3. Run the application:
 
     ```bash
-    cp committeeoversight/local_settings.example.py committeeoversight/local_settings.py
+    docker-compose up
     ```
 
-5. Start a `hearings` database with a scrape from the [hearings repo](https://github.com/datamade/hearings) or a data dump from a pal. If you're following the latter path and have an archival format `hearings.dump` file (see [documentation](https://www.postgresql.org/docs/10/app-pgrestore.html)) in the root of this project, restore it by running:
+3. The root of this repository has a file named `hearings.dump`, a sample set of hearings data for use in local development. With `docker-compose up` still running, open a new terminal tab and run:
 
     ```bash
-    pg_restore -C -j4 --no-owner hearings.dump | psql
+    docker exec committeeoversight-postgres pg_restore -C -j4 --no-owner -U postgres -d hearings /app/hearings.dump
```

Comment:
The [Docker docs](https://docs.docker.com/compose/reference/exec/) say that `docker-compose exec` is the equivalent of `docker exec`, but the latter runs fine while the former throws this error: 

```
ERROR: No such service: committeeoversight-postgres
```

When I run `docker-compose ps` to see the list of running services, though, `committeeoversight-postgres` is there: 

```
           Name                          Command                  State                Ports
-----------------------------------------------------------------------------------------------------
committeeoversight            python manage.py runserver ...   Up             0.0.0.0:8000->8000/tcp
committeeoversight-postgres   docker-entrypoint.sh postgres    Up (healthy)   0.0.0.0:32001->5432/tcp
```

I'm not sure why they're behaving differently—I used `docker exec` because that's the method in [this post](https://simkimsia.com/how-to-restore-database-dumps-for-postgres-in-docker-container/).

Reply:
ah yes! you need to use the service name, not the container name, in `docker-compose`. like `docker-compose run --rm postgres pg_restore...`